app/models/jsonld_record.rb

```diff
@@ -72,7 +72,17 @@ def cleaned_attributes
     end
 
     def outbound_graph
-      # TODO: Convert MODS/MARC to JSON-LD instead of using json service
-      @outbound_graph ||= RDF::Graph.load("https://bibdata.princeton.edu/bibliographic/#{bib_id}/jsonld") # FIXME: find IU equivalent link
+      @outbound_graph ||= generate_outbound_graph
```

Comment:
I'm nagged by the feeling that this good work should be factored out, and the result fed back upstream, so that sites can provide their own graph loaders and just configure in the one they like.